drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/enrollment_attribute_calculations.rb

```diff
@@ -84,12 +84,13 @@ module EnrollmentAttributeCalculations
       ].freeze
 
       def household_type(enrollment)
```

Comment:
Are they not interested in Children only households, or is this an accident?

Reply:
Just an attempt to protect the identities a bit further of children only households since this will eventually be published.